common/autoware_universe_utils/include/autoware/universe_utils/ros/diagnostics_module.hpp

```diff
@@ -59,6 +59,6 @@ void DiagnosticsModule::add_key_value(const std::string & key, const std::string
 template <>
 void DiagnosticsModule::add_key_value(const std::string & key, const bool & value);
```

Comment:
Please remove these declarations because they do not make sense unless you declare them as "extern template" and define them in cpp files as specializations.

Reply:
How about this? https://github.com/autowarefoundation/autoware.universe/pull/9714/commits/c981570b25bb48a1bd426b6e162cb531db3cc640